browser/base/content/newtab/newTab.xhtml

```diff
@@ -24,7 +24,7 @@
 </head>
 
 <body dir="&locale.dir;">
-  <iframe id="meep" src="file:///Users/msamuel/Projects/mozilla-source/remote-newtab/newTab.html"></iframe>
+  <iframe id="meep"></iframe>
```

Comment:
While we're here, let's give this a better name than `meep` :P how about `remote-newtab`?